recordedit/form.controller.js

```diff
@@ -104,6 +104,29 @@
             $log.info(response);
         }
 
+
+        /*
```

Comment:
@karlcz  Ok, I will make the changes, but will keep separates switch cases, so that in future if we want to transform values based on some other criteria , we can just add one more `if` statement to that `switch case`. Does that works ?